### PR TITLE
webhost updates

### PR DESCRIPTION
**Pull request by v-vishalr**
### Issue describing the changes in this PR

resolves #issue_for_this_pr

### Pull request checklist

* [ ] My changes **do not** require documentation changes
  * [ ] Otherwise: Documentation issue linked to PR
* [ ] My changes **do not** need to be backported to a previous version
  * [ ] Otherwise: Backport tracked by issue/PR #issue_or_pr
* [ ] I have added all required tests (Unit tests, E2E tests)

### CODE REVIEW

File: src/Azure.Functions.Cli/Azure.Functions.Cli.csproj [hunk 1]
```diff
@@ -156,7 +156,7 @@
     <PackageReference Include="Microsoft.ApplicationInsights" Version="2.18.0" />
     <PackageReference Include="Microsoft.AspNetCore.DataProtection" Version="2.2.0" />
     <PackageReference Include="Microsoft.Azure.DurableTask.AzureStorage.Internal" Version="1.4.0" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Script.WebHost" Version="4.0.1" />
+    <PackageReference Include="Microsoft.Azure.WebJobs.Script.WebHost" Version="4.1.3" />
```
Comment (ejizba) [inline src/Azure.Functions.Cli/Azure.Functions.Cli.csproj:159]: Hi, you need to make sure the workers match the v4.1.3 tag in the host repo. It appears java is the only one that doesn't match

https://github.com/Azure/azure-functions-host/blob/v4.1.3/src/WebJobs.Script/WebJobs.Script.csproj#L52

Comment (v-vishalr) [reply src/Azure.Functions.Cli/Azure.Functions.Cli.csproj:159]: @VpOfEngineering 
Can you please take a look at this

Comment (amamounelsayed) [reply src/Azure.Functions.Cli/Azure.Functions.Cli.csproj:159]: Thank you @ejizba, @shreyas-gopalakrishna  please can you confirm?

Comment (shreyas-gopalakrishna) [reply src/Azure.Functions.Cli/Azure.Functions.Cli.csproj:159]: @v-vishalr  For v4.1.3 we want the 2.0.0 java worker

Comment (v-vishalr) [reply src/Azure.Functions.Cli/Azure.Functions.Cli.csproj:159]: @VpOfEngineering updated java worker to 2.0.0

Comment (v-vishalr) [reply src/Azure.Functions.Cli/Azure.Functions.Cli.csproj:159]: @VpOfEngineering  updated

**Comment by VpOfEngineering** [comment]
@ejizba and @michaelpeng36 - moving forward for each release, should the instruction just be to update these references to match the host repo's?

**Review by ejizba — APPROVED** [review]

**Comment by ejizba** [comment]
I'm working on improvements to this area, possibly involving a script to validate and/or update versions. I can help update the instructions in time for the next release

**Pull request merged**